pkg/shed/db.go

```diff
@@ -74,83 +87,359 @@ func NewDB(path string, logger logging.Logger) (db *DB, err error) {
 		}
 	}
 
-	// Create a quit channel for the periodic metrics collector and run it
-	db.quit = make(chan struct{})
-
+	db.metrics.DBOpenCount.Inc()
+	logger.Debugf("Opened DB with path : %s, valueThreshold : %d, valueLogMaxEntries : %d",
+		path, DefaultValueThreshold, DefaultValueLogMaxEntries)
 	return db, nil
 }
 
-// Put wraps LevelDB Put method to increment metrics counter.
+// Put inserts the given key and value in to badger.
 func (db *DB) Put(key []byte, value []byte) (err error) {
-	err = db.ldb.Put(key, value, nil)
-	if err != nil {
-		db.logger.Debugf("failed to insert in to DB. Error : %s",  err.Error())
-		db.metrics.PutFailCounter.Inc()
-		return err
-	}
-	db.metrics.PutCounter.Inc()
-	return nil
+	db.logger.Tracef("Put value len : %d for key %s", len(value), string(key))
```

Comment:
this will flood the logs on `trace` (imagine thousands of these lines when you upload and sync). i think this is not needed